app/views/councillor_contributions/_instructions_body.md

```diff
@@ -1,4 +1,4 @@
-You should be able to find the information we need from the <%= @authority.full_name %> website.
+You should be able to find the information we need from the <%= link_to(@authority.full_name, @authority.website_url) %> website.
```

Comment:
What if we don't know the website URL for this council?

Could be a good use for 'link_to_if' http://api.rubyonrails.org/v4.2.7.1/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to_if

Note in those docs, you can probably replace the hash of controller/action options with a URL string—I think :)